cmds/console/console.go

```diff
@@ -32,7 +32,7 @@ func main() {
 
 	a := flag.Args()
 	if len(a) == 0 {
-		a = []string{"/ubin/rush"}
+		a = []string{"/ubin/elvish"}
```

Comment:
could we make this ubin/sh and fix sh to be a symlink?

Reply:
we already have defaultsh. Does /ubin/elvish work for bb mode?

Reply:
This is not as simple as creating a symlink because bb reads from argv[0] and would have to deference the sh symlink.

Right now, we have a special case for defaultsh inside of init.